XmlFormatter/src/Update/LocalVersionReciever.cs

```diff
@@ -0,0 +1,41 @@
+﻿using PluginFramework.EventMessages;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using XmlFormatterModel.Update;
+using XmlFormatterModel.Update.Strategies;
+
+namespace XmlFormatter.src.Update
+{
+    class LocalVersionReciever : IVersionRecieverStrategy
+    {
+        public event EventHandler<BaseEventArgs> Error;
+
+        public Task<IRelease> GetLatestRelease()
+        {
+            return default;
+        }
+
+        public Task<List<IRelease>> GetReleases()
+        {
+            return default;
+        }
+
+        public async Task<Version> GetVersion(IVersionConvertStrategy convertStrategy)
+        {
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            string versionString = "0.0.0";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'versionString'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)